src/main/java/io/seqera/tower/cli/responses/RunList.java

```diff
@@ -0,0 +1,42 @@
+package io.seqera.tower.cli.responses;
+
+import io.seqera.tower.cli.utils.TableList;
+import io.seqera.tower.model.ListWorkflowsResponseListWorkflowsElement;
+
+import java.io.PrintWriter;
+import java.util.List;
+
+public class RunList extends Response {
```

Comment:
The command `towr --json runs list` fails on binary CLI.

Reply:
Can't replicate this error locally.

Reply:
Working!